src/clofidence/utils.clj

```diff
@@ -11,7 +11,9 @@
          (let [def-init-form (last form)]
            (when (seq? def-init-form)
              (when-let [fdi (first def-init-form)]
-               (#{"fn" "fn*"} (name fdi))))))))))
+               (if (seq? fdi)
```

Comment:
The purpose of def-fn? is just to determine if the form we are giving to it has the shape of `(def xxx (fn [..] ...))` so I would fix it like : 
```clojure
(defn def-fn? [form]
  (boolean
   (when (seq? form)
     (when-let [fsymb (first form)]
       (when (= "def" (name fsymb))
         (let [def-init-form (last form)]
           (when (seq? def-init-form)
             (when-let [fdi (first def-init-form)]
               (and (symbol? fdi)
                    (#{"fn" "fn*"} (name fdi)))))))))))
```